src/shared/messaging/test/port-util-test.js

```diff
@@ -54,6 +57,7 @@ describe('port-util', () => {
           frame1,
           frame2,
           type,
+          requestId,
```

Comment:
I would remove these `requestId` properties. They are not checked and a message is valid without them.